src/main/java/Interpreter.java

```diff
@@ -0,0 +1,23 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Interpreter {
+
+
+    public List<String> getOperands(String inputExpression) {
+        return checkFirstValue(inputExpression);
+    }
+
+    public List<String> getOperators(String inputExpression) {
+        return new ArrayList<>(List.of(inputExpression.split("[0-9]+")));
+    }
+
+    private List<String> checkFirstValue(String inputExpression) {
```

Comment:
This method not only checks first value but also extracts operands. Extract checking the first value to separate method.

Reply:
already fixed it :)